uint32[] id_list
float32[] angle_list
float32 speed
float32 param
uint32 mode